scripts/vyatta-clear-squid

```diff
@@ -15,8 +15,8 @@ if test -n "$PID"; then
    mv $CACHE $CACHE.old
    mkdir $CACHE
    chown -R proxy:proxy $CACHE
-   squid3 -z
-   /etc/init.d/squid3 start
+   squid -z
+   /etc/init.d/squid start
```

Comment:
Shouldn't we be using systemctl for it now? It's quite likely that it will be changed from a legacy SysV init script to a native systemd unit soon.

Reply:
Done